data/wesionary-team.json

```diff
@@ -0,0 +1,16 @@
+{
+    "name": "wesionaryTEAM Co. Ltd.",
+    "establishedAt": 1580667300,
+    "location": "KVD Complex 2F, Balkumari - 08, Lalitpur",
+    "website": "https://en.wesionary.team/",
+    "email":"hr@wesionary.team",
+    "phone":"",
+    "socials": {
+      "facebook": "https://www.facebook.com/wesionaryteam",
+      "instagram": null,
+      "github": "https://github.com/wesionaryTEAM",
+      "linkedin": "https://www.linkedin.com/company/wesionary-team/",
+      "twitter": null,
+      "youtube":null
```

Comment:
```suggestion
      "youtube": null
```

Reply:
Youtube link for the company: https://www.youtube.com/channel/UCzmXl7zESI7CIAyEYkRu8Nw